Fix default multiselect height

skjnldsv opened the pull request:
To review before: https://github.com/nextcloud/nextcloud-vue/pull/1578

Switched to standard positioning to make sure the container take the content's height.
Allows design like this

![Peek 26-11-2020 09-38](https://user-images.githubusercontent.com/14975046/100327306-6daf3e80-2fcb-11eb-817c-904227d29f1d.gif)

PVince81 commented:
when testing this branch with the SharingInput on server, the dropdown looks broken:

<img width="691" alt="image" src="https://user-images.githubusercontent.com/277525/99559808-76998200-29c5-11eb-9497-9a1145ba4d00.png">

nextcloud-vue master is fine

I guess this will probably require more adjustments for consumers.

skjnldsv commented:
> when testing this branch with the SharingInput on server, the dropdown looks broken:

As mentionned: 

> To review before: #1578

Please review #1578 before :stuck_out_tongue: 
There are some fixes for this bug there

PVince81 commented:
> > when testing this branch with the SharingInput on server, the dropdown looks broken:
> 
> As mentionned:
> 
> > To review before: #1578
> 
> Please review #1578 before stuck_out_tongue
> There are some fixes for this bug there

ah I was confused and I thought "To review before X" meant that this PR here needs to be reviewed before X, not the other way around. I usually add an entry "- [ ] REQUIRES: X" so people know there's another dependency to look at first.

skjnldsv commented:
> ah I was confused and I thought "To review before X"

Right, englisch is hart! :see_no_evil: 
Now I see how it can be read both ways :stuck_out_tongue_closed_eyes:, sorry for that! :hugs:

PVince81 commented:
#1578 is merged, rebase ?

skjnldsv commented:
![Peek 26-11-2020 09-38](https://user-images.githubusercontent.com/14975046/100327227-4c4e5280-2fcb-11eb-8ab8-e63efb97cfed.gif)
Rebased and fixed Multiselect inner height (with docs)

PVince81 commented on src/components/ListItemIcon/ListItemIcon.vue:
```diff
@@ -189,8 +198,11 @@ export default {
 
 		cssVars() {
+			// Don't use margin to calculate the height if noMargin
+			const margin = this.noMargin ? 0 : this.margin
```
why not take "margin='0'" as property for disabling ? probably we don't want people supplying custom margins I guess ?

skjnldsv replied:
> probably we don't want people supplying custom margins I guess ?

Yep :/

PVince81 commented on src/components/Multiselect/Multiselect.vue:
```diff
@@ -110,33 +111,34 @@ By specifying `:user-select="true"`, you can benefit from a fully formatted layo
+
+// Building fake data for the docs
+const options = ['admin', 'user1', 'user2', 'guest', 'group1']
+const formatedOptions = options.map(item => {
```
minor typo => "formattedOptions"

PVince81 reviewed (approved):
:+1: see minor comments
